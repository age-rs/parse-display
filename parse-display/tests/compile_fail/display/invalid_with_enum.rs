use parse_display::Display;

#[derive(Display)]
enum X {
    #[display("{0}")]
    A(#[display(with = unknown)] u32),
    B,
}

fn main() {}
