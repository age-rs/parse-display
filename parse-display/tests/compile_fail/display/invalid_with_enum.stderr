error[E0425]: cannot find value `unknown` in this scope
 --> tests/compile_fail/display/invalid_with_enum.rs:6:24
  |
6 |     A(#[display(with = unknown)] u32),
  |                        ^^^^^^^ not found in this scope
